src/ontology/cob.Makefile

```diff
@@ -103,6 +103,12 @@ $(COB_ANNOTATIONS): $(TMPDIR)/cob-annotations.ttl
 	--annotation owl:versionInfo $(TODAY) \
 	--output $@
 
+$(IMPORTDIR)/ro_import.owl: $(MIRRORDIR)/ro.owl $(IMPORTDIR)/ro_terms_combined.txt
+	if [ $(IMP) = true ]; then $(ROBOT) query -i $< --update ../sparql/preprocess-module.ru \
+		extract -T $(IMPORTDIR)/ro_terms_combined.txt --copy-ontology-annotations true --force true --individuals exclude --method BOT \
+		remove --base-iri http://purl.obolibrary.org/obo/COB_ --axioms internal --preserve-structure false --trim false \
```

Comment:
This is the custom row: removing all COB related axioms from RO, but otherwise pulling in logical dependencies.